components/currentAgenda.tsx

```diff
@@ -34,14 +41,30 @@ export const CurrentAgenda: React.FC<CurrentAgendaProps> = ({
         sessionsUrl={sessionsUrl}
         acceptingFeedback={acceptingFeedback}
         feedbackLink={feedbackLink}
-        render={(agendaSessions, onSelect) => {
+        render={(agendaSessions, nextSessionGroup, onSelect) => {
           return (
             <AgendaProvider
               onSelect={onSelect}
               sessions={agendaSessions}
               sponsors={sponsors}
               rooms={['Theatre', 'RR5', 'M6', 'M7', 'M8', 'M9']}
             >
+              {Conference.ShowNextSessions && nextSessionGroup && nextSessionGroup.sessions.length > 0 && (
```

Comment:
Should we also add a dates.IsOnTheDay (or whatever it is) check too?

Reply:
that logic is resolved within useSessionGroup